### DISCUSSION TITLE
Problem with printer

### DISCUSSION THREAD

**Discussion by RodriRARG** [aria1th/litematica-printer#38] [Q&A] [answered]
Hi! im having this problem while trying to use printer. 
[Server thread/WARN]: Ignoring UseItemOnPacket

I tried with my client and others friends clients

Server is fabric with lithium

**Comment by aria1th** [comment, marked as answer]
can you post in issues? I guess its related to accurateblockplacement - mojang implemented anticheat, so you have to turn off those features if you don't have updated carpet extra on server. Use fakeRotation instead.
